src/db/results.rs

```diff
@@ -0,0 +1,47 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Result types for database methods.
+
+use std::collections::HashMap;
+
+use serde::{Deserialize, Deserializer, Serialize, Serializer};
+
+pub type GetCollections = HashMap<String, u64>;
+pub type GetCollectionCounts = HashMap<String, u64>;
+pub type GetCollectionUsage = HashMap<String, u32>;
+pub type GetQuota = Vec<u32>;
```

Comment:
Specified units for usage and quotas in the API are kilobytes, so I figured `u32` was plenty big enough for these two?

Ref: https://mozilla-services.readthedocs.io/en/latest/storage/apis-1.5.html#general-info